support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -83,17 +83,14 @@ export const tests: Tests = {
     targetPage: contributionsLandingPageMatch,
   },
 
-  paymentSecurityDesignTest: {
+  paymentSecurityDesignTest1B: {
```

Comment:
could we call this something more descriptive? what makes this 1b? i like long, descriptive names :)

Reply:
👍